### PR TITLE
Update guest-access-checklist.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2482
Added note.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@LanaChin  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/guest-access-checklist.md [hunk 1]
```diff
@@ -43,6 +46,9 @@ For more details, see [What the guest experience is like](guest-experience.md) a
 
 [!INCLUDE [guest-vs-external-access](includes/guest-vs-external-access.md)]
 
+> [!NOTE] 
+> Currently, Teams does not support the guest inviter role. At a minimum the members can invite toggle must be set to "Yes" for guest access to work in Teams. If you set members can invite to "No" and then enable guest access in Office 365 Groups and Microsoft Teams, admins can control guest invitations to your directory. After guests are in the directory, they can be added to teams by non-admin members who are team owners.
```
Comment (JohanFreelancer9) [inline Teams/guest-access-checklist.md]:
```suggestion
> Currently, Microsoft Teams does not support the guest inviter role. At a minimum the "members can invite" toggle must be set to "Yes" for guest access to work in Microsoft Teams. If you set "members can invite" to "No" and then enable guest access in Office 365 Groups and Microsoft Teams, admins can control guest invitations to your directory. After guests are in the directory, they can be added to teams by non-admin members who are team owners.
```

**Comment by LanaChin** [comment]
@SharS - over to you

**Pull request merged**